conda_build/cli/main_convert.py

```diff
@@ -124,6 +124,10 @@ def parse_args(args: Sequence[str] | None) -> tuple[ArgumentParser, Namespace]:
 
 
 def execute(args: Sequence[str] | None = None) -> int:
+    from .logging import init_logging
```

Comment:
Yeah, this should be pretty cheap to import. I only inlined it since we've been inlining as much as possible within `conda.cli` as we work to lower the import side-effects.